toolchain/check/context.cpp

```diff
@@ -1261,9 +1261,16 @@ class TypeCompleter {
 };
 }  // namespace
 
-auto Context::TryToCompleteType(SemIR::TypeId type_id) -> bool {
+auto Context::TryToCompleteType(SemIR::TypeId type_id, SemIRLoc loc) -> bool {
+  return TypeCompleter(*this, loc, nullptr).Complete(type_id);
+}
+
+auto Context::CompleteTypeOrCheckFail(SemIR::TypeId type_id) -> void {
   // TODO: We need a location here in case we need to instantiate a class type.
```

Comment:
Is this TODO still relevant?

Reply:
Huh, I thought I deleted that. Removed now. Thanks!